### PR TITLE
Java path Isolation and Python execution code added to new score modules

### PR DESCRIPTION
**Pull request by genadi-hp**
1. Added support for java path isolation in new score module. Not used still from ClaoudSlang
2. Python script execution code moved to the score

Signed-off by: Rabinovich Genad <genadi@hpe.com>

### CODE REVIEW

File: dependency-management/pom.xml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    </parent>
+
+    <artifactId>dependency-management</artifactId>
+    <packaging>pom</packaging>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-beans</artifactId>
+            <version>4.2.5.RELEASE</version>
```
Comment (Bonczidai) [inline dependency-management/pom.xml]: do we want different version here than the one declared in parent pom?

File: runtime-management/pom.xml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>runtime-management</artifactId>
+    <packaging>pom</packaging>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-beans</artifactId>
+            <version>4.2.5.RELEASE</version>
```
Comment (Bonczidai) [inline runtime-management/pom.xml]: do we want different version here than the one declared in parent pom?

File: dependency-management/pom.xml (hunk 1 above)
Comment (genadi-hp) [inline dependency-management/pom.xml]: As we already discussed - we will leave it for now here. Once you decide we deliver new modules always as part of score I will move it to the parent pom.xml

File: runtime-management/pom.xml (hunk 2 above)
Comment (genadi-hp) [inline runtime-management/pom.xml]: As we already discussed - we will leave it for now here. Once you decide we deliver new modules always as part of score I will move it to the parent pom.xml

**Pull request merged**